virttest/shared/cfg/guest-os/Windows.cfg

```diff
@@ -61,6 +61,7 @@
     # To avoid Windows OS bug, add this workaround.
     # https://gitlab.com/qemu-project/qemu/-/issues/2008
     default_bios:
+        no Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8 Host_RHEL.m9.u0 Host_RHEL.m9.u1 Host_RHEL.m9.u2 Host_RHEL.m9.u3
```

Comment:
Hi @leidwang, thanks for the fix :+1: I run the test, and I think we need to change a bit:
```
        ! Host_RHEL.m6, Host_RHEL.m7, Host_RHEL.m8, Host_RHEL.m9.u0, Host_RHEL.m9.u1, Host_RHEL.m9.u2, Host_RHEL.m9.u3:
            machine_type_extra_params += "smbios-entry-point-type=32"
```
(Without this change it just failed as before.)

Reply:
Using the change from above, the installation of Win2012 guest just passed:
```
 (1/6) Host_RHEL.m8.u10.v0.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: STARTED
 (1/6) Host_RHEL.m8.u10.v0.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: PASS (637.55 s)
```

Reply:
Sure, updated.Would you please check it again?Thanks!

Reply:
Great, thanks! LGTM now. 

I am running installations of 6 Windows guests right, I already mentioned Win2012, and Win2019 installation also looks good (qemu cmd runs, so it definitely resolved the problem with `smbios-entry-point-typ`). I think we are good here with the first tests passing. The failures were always exactly the same.